### PR TITLE
Gh 1371 fix tracking

### PR DESCRIPTION
**Pull request by sche**
Handles #1371, #1372, #1373, #1374

### CODE REVIEW

File: Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift [hunk 1]
```diff
@@ -28,10 +28,13 @@ struct EmailSupportViewController: UIViewControllerRepresentable {
...
         let version = "v\(marketingVersion) (\(buildNumber))"
         let safe = try? App.shared.coreDataStack.viewContext.fetch(Safe.fetchRequest().selected()).first
+        let address = safe?.address ?? "None"
+        let chain = safe != nil ? "\(safe!.chain!.name!) (\(safe!.chain!.id!))" : "None"
 
         let message = """
         Gnosis Safe \(version)
-        Safe address: \(safe?.address ?? "None")
+        Safe address: \(address)
+        Chain: \(chain)
```
Comment (biafra23) [inline Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift:37]: We (the Android team) decided to also include the id in braces as in: Rinkeby (4)

Comment (sche) [reply Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift:37]: it is included

Comment (sche) [reply Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift:37]: Here I asked a related question https://github.com/gnosis/safe-android/pull/1627#discussion_r685073797

**Review by biafra23 — APPROVED** [review]
LGTM

**Pull request merged**